docs/configuration/document-settings.md

```diff
@@ -1,186 +0,0 @@
----
```

Comment:
It got moved to `in-document-settings.md`. I renamed the file because it didn't match the actual title of the document, which was confusing and/or a bug.